lib/cli/interactive-setup/service.js

```diff
@@ -101,9 +101,10 @@ const resolveProjectNameInput = async (options, workingDir) => {
 
 module.exports = {
   isApplicable({ options, serviceDir }) {
-    if (serviceDir && (options.name || options['template-path'])) {
+    const notApplicableOptions = new Set(['name', 'template-path', 'template', 'template-url']);
+    if (serviceDir && Object.keys(options).some((key) => notApplicableOptions.has(key))) {
       throw new ServerlessError(
-        'Cannot setup a new service when being in context of another service ("--name" and "--template-path" options cannot be applied)',
+        'Cannot setup a new service when being in context of another service ("--name", "--template-path", "--template" and "--template-url" options cannot be applied)',
```

Comment:
nit: we inject names into error message via `[...notApplicableOptions].join('", --"')`, it'll be less error prone